Automattic/wp-calypso
# Strip trailing zeros from plan prices

#### Changes proposed in this Pull Request

* Reconfigure `formatCurrency` call to strip leading zeroes from plans prices.

#### Testing instructions

- Visit the plans grid at https://calypso.live/new?branch=update/strip-zeroes-from-price.
- Make sure the prices look exactly like they do in `/start`.
